std/variant.d

```diff
@@ -1234,6 +1234,14 @@ public:
     assert(a[5] == 7);
 }
 
+@safe unittest
+{
+    alias Var = VariantN!(24);
+
+    Var a;
+    assert(a.sizeof == 24 + (void*).sizeof);
```

Comment:
```d
@safe unittest
{
    static assert(VariantN!(24).sizeof == 24 + (void*).sizeof);
    static assert((VariantN!(24)[4]).sizeof == ...);
}
```

Reply:
Addressed in c163aee
I copied the test below it so the alias remained. Sorry for the inconvenience :sweat_smile: